### DISCUSSION TITLE
Clicked the "Continue" button then give me "We require applicants to use a school-issued email address to apply"

### DISCUSSION THREAD

**Discussion by zhutianci1998** [community/community#72878] [GitHub Education]
### Hi everyone,
almost same discussions:
https://github.com/orgs/community/discussions/48005
https://github.com/orgs/community/discussions/47719

Clicked the "Continue" button then give me "We require applicants to use a school-issued email address to apply". It seems like github database doesn't have my university info.

<img width="883" alt="截屏2023-10-30 00 08 04" src="https://github.com/community/community/assets/134049661/09cef71f-bbcd-48aa-acb3-8d9d0d53b442">

I enabled [two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa), I do have a edu email which was provided by my school and I am a student.  In Edge, when i clicked the button but the page refreshed and clean all my info I filled, then give me this info. In chrome, the button just became grey, then the page freezed.
<img width="926" alt="截屏2023-10-29 22 36 28" src="https://github.com/community/community/assets/134049661/ad727f3a-aa39-40a5-aa6f-bfa72b264dfc">

**Comment by Naman-B-Parlecha** [comment]
Maybe  select the option from the list provided by GitHub instead of write the entire word or something

**Reply by zhutianci1998** [reply]
![image](https://github.com/community/community/assets/134049661/dfff7792-a872-4a8d-8b0b-32218f319ad6)
I actually selected the option from the list provided by GitHub.

**Reply by zhutianci1998** [reply]
I updated my question.

**Comment by Naman-B-Parlecha** [comment]
Okay u have a email provided by ur uni and want to use it 

Check this video it will help you out ig = 
https://youtu.be/iyThW3RvZpk?si=oc3iogy9iOQf_hzq

Check the comments section if any some error pops up and the discussion is always there

**Reply by zhutianci1998** [reply]
![image](https://github.com/community/community/assets/134049661/5998662a-3ce3-447a-ae8a-7178afc08c98)
I know the reason, GitHub can't recognize my university's new email address 'stu.yun.edu.cn'
